.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,67 @@
+on:
+  push:
+    tags:
+      - "v*" # Push events to matching `v*` version srings. e.g. v1.0, v20.15.10
+
+name: Create and Publish Release
+
+jobs:
+  build:
+    name: Create and Publish Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install release dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade setuptools wheel twine
+
+      - name: Get release notes
+        id: release_notes
+        run: |
+          # By default, GH Actions checkout will only fetch a single commit.
+          # For us to extract the release notes, we need to fetch the tags
+          # and tag annotations as well.
+          # https://github.com/actions/checkout/issues/290
+          git fetch --tags --force
+          TAG_NAME=${GITHUB_REF/refs\/tags\//}
+          CHANGELOG=$(git tag -l --format='%(contents)' $TAG_NAME)
+          # we have to escape all new line characters and carriage
+          # returns before passing them into set-output otherwise the
+          # body content of the release will only contain the first line
+          # https://github.com/actions/create-release/issues/38#issuecomment-640072234
+          CHANGELOG="${CHANGELOG//'%'/'%25'}"
+          CHANGELOG="${CHANGELOG//$'\n'/'%0A'}"
+          CHANGELOG="${CHANGELOG//$'\r'/'%0D'}"
+          echo "::set-output name=changelog::$CHANGELOG"
+
+      - name: Create GitHub release
+        id: create_release
+        # create-release@v1 doesn't support the body arg so we use
+        # create-release@master instead.
+        uses: actions/create-release@master
```

Comment:
I only see 3 commits to master since the [`v1` release](https://github.com/actions/create-release/releases/tag/v1) commit.  `body` should be supported as of that commit.  I am using `actions/create-release@v1` with the `body` field defined using a string literal in all of my projects.  May be worth re-pinning this @ `v1` in case they decide to push a new major release at some stage.

Reply:
Confirmed: https://github.com/actions/create-release/blob/0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e/action.yml#L11

Reply:
There's also https://github.com/actions/create-release/blob/0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e/action.yml#L14 -- maybe you want to pipe the annotation text to a file instead of doing env var acrobatics?

Reply:
@madig agreed, it's much better. Thank you!